mesh_startup.scd

```diff
@@ -1,12 +1,19 @@
-// [OPTIONAL]
-// load remote server options for the mesher
-"mesher_server_options.scd".loadRelative;
+// LAN session with shared server
+// Run on each local client:
+~serverIP = "192.168.0.6";
+~serverPort = 57110;
+"sessionConfig/mesher_server_options.scd".loadRelative;
+
+// join Utopia, setup chat, SynthDescRelay and CodeRelay
+"sessionConfig/setup_utopia.scd".loadRelative;
```

Comment:
I think there was a reason that I wanted to have this loaded after setting up the server options etc. I can't remember exactly. I do remember that the synthDescRelay does not retroactively publish changes to clients that join Utopia. i.e. If you aren't joined, and someone adds a SynthDef, you won't have it locally.
I didn't check whether adding a new synthdef syncs the existing SynthDesLib or just adds the new one.
In any case, we might want to revisit to find the best order for each of these includes.